snort/docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: snort_web_1
+      APP_PORT: 80
+
+  web:
+    image: ghcr.io/v0l/snort:v0.1.4
```

I see the newest version tag is v.0.1.3 here: https://github.com/v0l/snort/pkgs/container/snort

Are we just waiting on 0.1.4 to be pushed?

If you want to go with 0.1.3 you can use this sha for multi-arch:

```suggestion
    image: ghcr.io/v0l/snort:v0.1.3@sha256:bab2b88b09e7894152e530b4bee06df532cdff79389a60eedcc197675bc63fb6
```

---

Ah, I see that 0.1.4 is the umbrel release candidate. I'll wait until it is available.

---

I have tested new 0.1.4 image on arm64 and amd64. Can you please add the SHA256 hash:

```suggestion
    image: ghcr.io/v0l/snort:v0.1.4@sha256:734f74393e2560e625bad9e2485ccccfa6af53caab64a67d971380244cfc54dd
```